webpack.config.js

```diff
@@ -124,27 +124,13 @@ const reactNativeConfig = {
  *
  *   1. This package is designed for Node only and hence requires polyfills of Node libraries (e.g. `stream`, `path`) — webpack takes care of this for us.
  *   2. The package is not compatible with RequireJS and hence we don’t have any easy way to directly load it in our tests — the webpack bundle exposes it as a global named MochaJUnitReporter.
- *
- * - Node: The library uses optional chaining syntax, which is not supported by Node 12.
  */
 function createMochaJUnitReporterConfigs() {
   const dir = path.join(__dirname, 'test', 'support', 'mocha_junit_reporter');
 
   const baseConfig = {
     mode: 'development',
     entry: path.join(dir, 'index.js'),
-    module: {
-      rules: [
-        {
-          // The optional chaining syntax used by mocha-junit-reporter is not supported by Node 12 (see above). Hence, we transpile using Babel.
-          test: /\.js$/,
-          loader: 'babel-loader',
```

Comment:
I don't think we're using `babel-loader` or `@babel/preset-env` any more, can we remove these dependencies?

Reply:
True, will remove those, thanks!